census/src/main/java/io/grpc/census/CensusStatsModule.java

```diff
@@ -67,6 +69,7 @@
  * starts earlier than the ServerCall.  Therefore, only one tracer is created per stream/call and
  * it's the tracer that reports the summary to Census.
  */
+@Internal
 public final class CensusStatsModule {
```

Comment:
If these are public only for testing, then VisibleForTesting is appropriate. The Internal tells our users not to use it, the VisibleForTesting tells us not to use it.

Reply:
Alright, then we probably need both.